src/Network/Network/help/Get-AzNetworkManagerVerifierWorkspace.md

```diff
@@ -0,0 +1,341 @@
+NetworkManagerName : testNM
+Type               : Microsoft.Network/networkManagers/verifierWorkspaces
+SystemData         : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSSystemData
+SystemDataText     : {
+                       "CreatedBy": "test@email.com",
+                       "CreatedByType": "User",
+                       "CreatedAt": "2024-02-16T00:03:39.541236Z",
+                       "LastModifiedBy": "test@email.com",
+                       "LastModifiedByType": "User",
+                       "LastModifiedAt": "2024-02-16T00:24:13.1874766Z"
+                     }
+Id                 : /subscriptions//00000000-0000-0000-0000-000000000000/resourceGroups/testRG/providers/Microsof
+                     t.Network/networkManagers/testNM/verifierWorkspaces/testVerifierWorkspace9
+```
+Gets the network manager verifier workspace of name'testVerifierWorkspace9'.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+The resource name.
+
+```yaml
+Type: String
+Parameter Sets: NoExpand
+Aliases: ResourceName
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: True
+```
+
+```yaml
+Type: String
+Parameter Sets: Expand
+Aliases: ResourceName
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: True
+```
+
+### -NetworkManagerName
+The network manager name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: True
+```
+
+### -ProgressAction
+{{ Fill ProgressAction Description }}
+
+```yaml
+Type: ActionPreference
+Parameter Sets: (All)
+Aliases: proga
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The resource group name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: True
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### System.String
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.Network.Models.NetworkManager.PSVerifierWorkspace
+
+## NOTES
+
+## RELATED LINKS
```

Comment:
add related links to all help docs; these are the corresponding Set, Remove, etc cmdlets for the same resource type